run_kind_ci.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+KIND_CI_PROJECTS=("sriov-cni" "sriov-ib" "antrea" "ipoib" "network-operator" "antrea")
```

Comment:
antrea is duplicated, can you remove the duplication?

Reply:
fixed